jpa-crud/jpa-crud-java-sbt/README.md

```diff
@@ -0,0 +1,58 @@
+# Lagom Recipe: JPA CRUD  
+
+This recipe demonstrates, how you can use Play's [JPA API](https://www.playframework.com/documentation/2.6.x/JavaJPA) 
+for implement Lagom service with CRUD-oriented persistence.
+
+## About service
```

Comment:
This recipe is relying on a custom library which wraps Play JPA and makes it easier to use. It'd be good if this was mentioned upfront.

Reply:
Added the link to the library and some words in readme.